File: 20220727/모범답안/20220727_02.js/CountEvenEachNumber.js
/**
 * @param n {number}
 * @returns {number}
 */
function solution(n) {
  let result = 0

  while (1 <= n) {
    const i = Math.floor(n % 10)
    result += (i % 2 === 0) ? 1 : 0

    n /= 10
  }

  return result
}

export default solution

File: 20220727/모범답안/20220727_02.js/CountEvenEachNumber.py
import math


def solution(n):
    """
    :param n: number
    :return: number
    """

    result = 0

    while (1 <= n):
        i = math.floor(n % 10)
        result += 1 if i % 2 == 0 else 0

        n /= 10

    return result
